### DISCUSSION TITLE
adding new ink! tutorial (inetemdiate)

### DISCUSSION THREAD

**Discussion by midegdugarova** [PolkadotStudy/polkadot.study#27] [New Tutorials]
Hello, 
We (Astar team) would like to add a comprehensive tutorial on how to create a minting ink! smart contract and a fungible token contract. The tutorial dives into intermediate ink! as well as cross-contract calling,e2e tests and OpenBrush. 

Please let us know your opinion!

**Comment by niklasp** [comment]
Great! You would be the first parachain team to contribute a tutorial. There will be a dedicated section / page for each parachain which lists all tutorials on polkadot.study and anything else you want (written in markdown) similar tk tag pages atm:

https://polkadot.study/tutorials/tags/polkadot-js

To start, please submit a new issue with the 'new tutorial' template so we can discuss content. Also the grants we have are reserved for community devs. Are you okay with that?

**Comment by niklasp** [comment]
@midegdugarova any update here? Do you need help at submitting a PR?
